samples/IntegrationTestApp/MainWindow.axaml.cs

```diff
@@ -53,6 +56,22 @@ private void Pager_SelectionChanged(object? sender, SelectionChangedEventArgs e)
                 PagerContent.Child = page.CreateContent();
         }
 
+        private void OnPositionChanged(object? sender, PixelPointEventArgs e)
+        {
+            // HACK: Toggling the window decorations can cause the window to be moved off screen, 
+            // causing test failures. Until this bug is fixed, detect this and move the window
+            // to the screen origin. See #11411.
+            if (Screens.ScreenFromWindow(this) is { } screen)
+            {
+                var bounds = new PixelRect(
+                    e.Point,
+                    PixelSize.FromSize(ClientSize, DesktopScaling));
+
+                if (!screen.WorkingArea.Contains(bounds))
```

Comment:
Shouldn't only the position be verified instead of the bounds? This might cause an infinite loop if the window somehow doesn't fit into the working area,

Reply:
The problem is when the bottom of the window is moved below the taskbar so we need to verify the bounds (the problem is that this causes the taskbar to be clicked instead of the window).

I guess we should also add a check that the window size is smaller than the working area though to prevent such an infinite loop.